projects/storefrontapp-e2e-cypress/cypress/integration/smoke/currency.e2e-spec.ts

```diff
@@ -0,0 +1,54 @@
+const JPY_CURR = 'JPY';
+const USD_CURR = 'USD';
+const BASE_URL = Cypress.config().baseUrl;
+const CONTENT_CATALOG = 'electronics-spa';
+
+const changeCurrency = (currency: string) => {
+  cy.get('.SiteContext label')
+    .contains('Currency')
+    .parent()
+    .children('select')
+    .select(currency);
+};
+
+context('Currency change', () => {
+  let PRODUCT_URL_USD: string;
+  let PRODUCT_URL_JPY: string;
+  let PRODUCT_ID: string;
+
+  describe('on the product page', () => {
+    beforeEach(() => {
```

Comment:
This I would also move to set value on declaration

Reply:
done